_damon_dbgfs.py

```diff
@@ -65,7 +65,9 @@ def wops_for_target(target, target_has_pid):
         _damon.damos_action_pageout: 2,
         _damon.damos_action_hugepage: 3,
         _damon.damos_action_nohugepage: 4,
-        _damon.damos_action_stat: 5}
+        _damon.damos_action_lru_prio: 5,
+        _damon.damos_action_lru_deprio: 6,
+        _damon.damos_action_stat: 7}
```

Comment:
Shouldn't `_damon.damos_action_stat` be updated to `7` at least even if `damos_action_lru_[de]prio` is not need here?

Reply:
I don't think so, because DAMON debugfs interface uses `5` as the value for the stat action[1], unless I missed something.  Is I'm missing something, please let me know.

[1] https://git.kernel.org/pub/scm/linux/kernel/git/sj/linux.git/tree/mm/damon/dbgfs.c?h=damon/next-2023-06-06-23-25-on-v6.4.0-rc4-mm-unstable#n206

Reply:
I thought that the numbers should be the matched in `_damon.py` at https://github.com/awslabs/damo/blob/v1.8.5/_damon.py#L604-L611, but maybe not. Thanks for the correction.